exonum-java-binding/integration-tests/src/test/java/com/exonum/binding/test/BlockchainIntegrationTest.java

```diff
@@ -343,21 +337,21 @@ void getActualConfiguration() {
   }
 
   @Test
-  void getTransactionPool() throws Exception {
+  void getTransactionPool(TestKit testKit) throws Exception {
     TestService service = testKit.getService(TestService.SERVICE_ID, TestService.class);
     TransactionMessage message = constructTestTransactionMessage("Test message", testKit);
     RawTransaction rawTransaction = RawTransaction.fromMessage(message);
     service.getNode().submitTransaction(rawTransaction);
 
-    testKitTest((blockchain) -> {
+    testKitTest(testKit, (blockchain) -> {
       KeySetIndexProxy<HashCode> transactionPool = blockchain.getTransactionPool();
       assertThat(transactionPool.contains(message.hash()))
           .describedAs("pool=%s", transactionPool)
           .isTrue();
     });
   }
 
-  private void testKitTest(Consumer<Blockchain> test) {
+  private void testKitTest(TestKit testKit, Consumer<Blockchain> test) {
```

Comment:
Wouldn't it be better to keep testkit as a field if it’s needed to pass it between methods?

Reply:
I agree, thought that it was better the way it was